packages/wallet-contract/schema/v1/wallet-contract-documents.json

```diff
@@ -0,0 +1,125 @@
+{
+  "txMetadata": {
+    "type": "object",
+    "indices": [
+      {
+        "name": "ownerId",
+        "properties": [
+          {
+            "$ownerId": "asc"
+          }
+        ]
+      },
+      {
+        "name": "ownerIdAndCreatedAt",
+        "properties": [
+          {
+            "$ownerId": "asc"
+          },
+          {
+            "$createdAt": "asc"
+          }
+        ]
+      }
+    ],
+    "properties": {
+      "keyIndex": {
+        "type": "integer",
+        "minimum": 0,
+        "description": "The index of the owners identity public key used to derive the encryption key.",
+        "position": 0
+      },
+      "encryptionKeyIndex": {
+        "type": "integer",
+        "minimum": 0,
+        "description": "The secondary index used to derive the encryption key that is used to encrypt and decrypt encryptedData.",
+        "position": 1
+      },
+      "encryptedMetadata": {
+        "type": "array",
+        "byteArray": true,
+        "minItems": 32,
+        "maxItems": 4096,
+        "description": "encrypted metadata using AES-CBC-256",
+        "position": 2
+      }
+    },
+    "required": [
+      "keyIndex",
+      "encryptionKeyIndex",
+      "encryptedMetadata",
+      "$createdAt"
+    ],
+    "additionalProperties": false
+  },
+  "identityVerify": {
```

Comment:
What is it for? How is it related to the wallet?

Reply:
In the DashPay app, when a user requests a contested username, we allow the user to specify a social media post to "prove" an identity.

If two people wanted `bob` each can specify a url to a social media post that says something like "I am the real bob and my identity id is `Abs19....47k`."

Masternode owners using DashPay can view this see this URL in the username request details page of the Username Voting screen. 

This contact might not be the best place for it.  If it isn't, I am not sure if it should be in a new contract or the dashpay contract.

Reply:
Yeah, it sounds like something else. Let's remove this from this PR and think more about it?